build/reactable.js

```diff
@@ -510,7 +510,9 @@
                        return; // (continue)
                        }
                        */
-                    if (typeof(child.props) !== 'object') {
+                    if (child == null) {
```

Comment:
since that's kindof what's being done on the line following this, should i just collapse that into one `if` and remove the `console.warn`?